src/test/java/com/artipie/asto/bench/BenchmarkStorageTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/asto/LICENSE.txt
+ */
+package com.artipie.asto.bench;
+
+import com.artipie.asto.Content;
+import com.artipie.asto.Key;
+import com.artipie.asto.ext.PublisherAs;
+import com.artipie.asto.memory.InMemoryStorage;
+import java.util.HashMap;
+import java.util.Map;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test for {@link BenchmarkStorage}.
+ * @since 1.1.0
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+final class BenchmarkStorageTest {
```

Comment:
You forget about the test for `deleted` set

Reply:
@g4s8 it is impossible now to verify this scenario because `delete` operation is not implemented yet and `deleted` is unreachable from outside